### PR TITLE
Gotify icon in notification, and keep notification in notification history.

### PR DESCRIPTION
**Pull request by baduhai**
By adding the parameter `-h` to notify-send, you are able to specify a desktop file to accompany the notification. This has two effects:

1. The notification gets an application icon if your DE or notification daemon supports it.
2. The notification stays in the the DE's notification history (of course if supported by the DE).

In order for the notification to have an application icon, I had to bundle the pngs in the repo, and I could only find these sizes. I made sure to check the licensing, and while I couldn't find any licensing specific to the icon, since Gotify itself is MIT, that should also apply to the icon. I was a bit worried about the `.desktop` file showing up in users' menus, but with the `NoDisplay=true` tag, the app doesn't show up in menus. Here's the before and after with the app icon:

Screenshot_20210823_213418

Screenshot_20210826_233835

And here it is in the notification history:

Screenshot_20210826_234059

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,4 +10,15 @@ install:
 	# files in /usr/lib
 	install -d $(DESTDIR)$(PREFIX)/lib/gotify-dunst/
 	install main.py $(DESTDIR)$(PREFIX)/lib/gotify-dunst/
-	install gotify-dunst.conf $(DESTDIR)$(PREFIX)/lib/gotify-dunst/
\ No newline at end of file
+	install gotify-dunst.conf $(DESTDIR)$(PREFIX)/lib/gotify-dunst/
+
+	# files in /usr/share
+	install gotify-dunst.desktop $(DESTDIR)$(PREFIX)/share/applications
```
Comment (karmanyaahm) [inline Makefile:17]: Needs an
`install -d $(DESTDIR)$(PREFIX)/share/applications`

**Review by karmanyaahm — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (karmanyaahm) [reply Makefile:17]: Required because Arch packages start off with an empty directory. I think other package formats do the same.

**Comment by karmanyaahm** [comment]
Apart from the above thing, this change is very good!

**Comment by baduhai** [comment]
Commit submitted with requested change. Tbh this change of having notifications stay in the history is something I've been looking for a long time for my own projects, and only recently found out how to do.

**Comment by ztpnk** [comment]
Sorry for my late answer. Tbh I had to re-activate my old gotify-server because I don't use it at the moment (I would but I don't have use cases...) to test the changes.
For me this changes aren't really necessary because I'm just using a simple i3wm setup but I like the idea of having this features on a more powerful DE.

So thanks for the changes and @karmanyaahm for the review!
I like how this project grows and gets pull requests from time to time. Like I said I actually don't use it myself but I hope I find some cool use cases soon

**Pull request merged**